scripts/boston_cityhall_electricity.json

```diff
@@ -0,0 +1,35 @@
+{
+    "citation": "FILL",
+    "description": "FILL",
```

Comment:
We have to fill in the values that have FILL
Give a good description of the dataset

`"description": "Electric power load at City Hall (1 City Hall Square) measured every 15 minutes."`

Landing page for the dataset

`"homepage": "https://data.boston.gov/dataset/city-hall-electricity-usage",`

The link used to download the  csv file

`"url": "https://data.boston.gov/dataset/1b894599-21ff-478f-937d-653954977951/resource/f123e65d-dc0e-4c83-9348-ed46fec498c0/download/tmpvi7_mwih.csv"`

`"licenses": [ { ... "name": "PDDL"`